tests/core/test_model.py

```diff
@@ -1332,7 +1334,10 @@ def my_model(context, **kwargs):
 
 def test_python_model_depends_on() -> None:
     @model(
-        name="model_with_depends_on", kind="full", columns={'"COL"': "int"}, depends_on=["foo.bar"]
+        name="model_with_depends_on",
+        kind="full",
+        columns={'"COL"': "int"},
+        depends_on={'"foo"."bar"'},
```

Comment:
Do we expect users to provide normalized depends_on in the `@model` decorator?

Reply:
Nope we will normalize. Updating test. Thanks for catching.